osquery/tables/system/darwin/secureboot.mm

```diff
@@ -176,17 +184,120 @@ Status getSecureBootSetting(Row& row) {
   return Status::success();
 }
 
+Status getAarch64SecureBootSetting(Row& r) {
+  @autoreleasepool {
+    // BEWARE: Because of the dynamic nature of the calls in this function, we
+    // must be careful to properly clean up the memory. Any future modifications
+    // to this function should attempt to ensure there are no leaks (and test
+    // with ./tools/analysis/profile.py --leaks).
+    CFURLRef bundle_url = CFURLCreateWithFileSystemPath(
+        kCFAllocatorDefault,
+        CFSTR("/System/Library/PrivateFrameworks/SPSupport.framework"),
+        kCFURLPOSIXPathStyle,
+        true);
+
+    if (bundle_url == nullptr) {
+      return Status::failure("Error parsing SPSupport bundle URL");
+    }
+
+    CFBundleRef bundle = CFBundleCreate(kCFAllocatorDefault, bundle_url);
+    CFRelease(bundle_url);
+    if (bundle == nullptr) {
+      return Status::failure("Error opening SPSupport bundle");
+    }
+
+    CFBundleLoadExecutable(bundle);
```

Comment:
`CFBundleLoadExecutable` returns a boolean, perhaps makes sense to check the return value..?